src/services/expensesService.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const { getNewId } = require('../helpers/getNewId');
+
+let expenses = [];
+
+const resetExpenses = () => {
+  expenses = [];
+
+  return expenses;
+};
+
+const getAllExpenses = (query) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = query;
+
+  let filteredExpenses = expenses;
+
+  if (userId) {
+    filteredExpenses = expenses
+      .filter(expense => expense.userId === Number(userId));
```

Comment:
Move this and the rest of `Number(...)` conversions to controller level. Dealing with input data is the area of concern for the controller, not the service.